### PR TITLE
Lots of fixes, some redesigning, and a new feature as well.

### PR DESCRIPTION
**Pull request by PyroSamurai**
Fixed header, footer, etc.
Added the new Google+ link.
Revamped and validated donate page.
Various other fixes that I'm too tired to list.
Check the commits, I documented the changes extensively.
Here is a 90% zoom screenshot of the new donate page,
since I know you may be hesitant about changes to that page:
selection_053

### CODE REVIEW

**Comment by PyroSamurai** [comment]
I would appreciate a quick response to this pull since I'm getting 20 emails a day from github telling me that www.sourceoftales.org is already being used and I can't delete CNAME without it affecting the pull.

File: _includes/header.html [hunk 1]
```diff
@@ -1,16 +1,15 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="chrome=1">
+    <!--<meta http-equiv="X-UA-Compatible" content="chrome=1">-->
```
Comment (stefanbeller) [inline _includes/header.html]: why comment? why not delete?

Comment (PyroSamurai) [inline _includes/header.html]: Because I hadn't checked with any of you to see if that was okay or not.

**Comment by Ablu** [comment]
Hm. You should get one message per commit, regardless of how long we
need to review :) Github rebuilds only on new commits.

I will see if i have time later.

**Comment by bjorn** [comment]
Looked fine to me. But please try to make fewer commits next time. If you want to fix up something you already changed in a previous commit, you should amend the commit instead.

**Comment by PyroSamurai** [comment]
My apologies. Other than my mistakes, I was trying to document the changes well and split them up into easy-to-understand pieces to speed up the review process.

**Comment by bjorn** [comment]
No problem, thanks for the improvements!

**Pull request merged**